sdk/monitor/Azure.ResourceManager.Monitor/src/Generated/RestOperations/AzureMonitorWorkspacesRestOperations.cs

```diff
@@ -0,0 +1,646 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager.Monitor.Models;
+
+namespace Azure.ResourceManager.Monitor
+{
+    internal partial class AzureMonitorWorkspacesRestOperations
```

Comment:
@ArthurMa1978 This file did not get renamed through autorest. Is it alright to leave it this way since it is outside of the Models directory, or should I manually change it? I did try adding a rename-mapping initially, but this file was unaffected.

Reply:
This is an internal class, so I think it's ok to leave it as it is.